### PR TITLE
🛣 nav

### PR DESCRIPTION
**Pull request by blacroix**

### CODE REVIEW

File: app/src/main/java/fr/publicissapient/planningpoker/model/CardSuitType.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package fr.publicissapient.planningpoker.model
+
+sealed class CardSuitType {
+    companion object {
+        const val FIBONACCI_RED = "FIBONACCI_RED"
+    }
+}
```
Comment (jinqian) [inline app/src/main/java/fr/publicissapient/planningpoker/model/CardSuitType.kt]: If we are using sealed class, I think it's better this way:
```suggestion
sealed class CardSuitType {
    data class Fibonacci(val route: String) :  CardSuitType()
    data class TshirSize(val route: String) : CardSuitType()
}
```

Comment (blacroix) [reply app/src/main/java/fr/publicissapient/planningpoker/model/CardSuitType.kt]: Here a discriminent is needed in order to select appropriate type of card, Compose uses String only 🤔

Comment (blacroix) [reply app/src/main/java/fr/publicissapient/planningpoker/model/CardSuitType.kt]: Red is removed!

File: app/src/main/java/fr/publicissapient/planningpoker/ui/CardScreen.kt [hunk 2]
```diff
@@ -15,55 +14,58 @@ import androidx.compose.ui.drawLayer
...
 
 @Composable
 fun CardScreen(
-    @DrawableRes imageResId: Int,
-    description: AnnotatedString,
-    pointValue: Int,
-    color: Color,
-) =
+    cardSuit: String?,
+    cardId: Int?,
```
Comment (jinqian) [inline app/src/main/java/fr/publicissapient/planningpoker/ui/CardScreen.kt]: why nullable here? 🤔

Comment (blacroix) [reply app/src/main/java/fr/publicissapient/planningpoker/ui/CardScreen.kt]: Not anymore!

**Pull request merged**